invenio_app_ils/ui/backoffice/src/pages/LoanDetails/components/LoanActions/tests/LoanAction.test.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import LoanAction from '../LoanActions';
+
+describe('LoanAction', () => {
+  it('renders initial state', () => {
+    const props = {
+      data: {},
+      onAction: () => 'onAction',
+      actions: {},
+    };
+    const component = shallow(<LoanAction {...props} />);
```

Comment:
I guess you are aware, but this is also for @topless and @kprzerwa, `shallow` will not call `componentDidMount` and `componentDidUpdate` of your React component, often used. 
If you use that hooks in your component, and you need to test the full behaviour of the component, you have to use `mount` instead of `shallow`... just warning because I already had headaches with this :)
https://stackoverflow.com/questions/38710309/when-should-you-use-render-and-shallow-in-enzyme-react-tests

Reply:
Also there is the render function that just renders all the children components of your component which is the case if you want to tests your final rendered template.